Makefile

```diff
@@ -19,5 +19,6 @@ coverage:
 	$(PYTESTS) skimage --cov=skimage
 
 html:
-	pip install -q sphinx pytest-runner sphinx-gallery
+	pip install -q -r requirements/docs.txt
+	rm -rf doc/build
```

Comment:
Not sure about this line. I often partially rebuild the docs, especially, when working on a specific example.

Reply:
ah, right, this isn't used only in releases. =) I'll add doc/build to make clean and I'll update the release instructions. I think it's important to do a clean build (since this is going to be the official documentation essentially forever), but you're right that it shouldn't be done always.